Play store text and image updates (#4448)
home-assistant/android: fastlane/metadata/android/en-US/full_description.txt

- Discover automatically and quickly configure new devices - such as Philips Hue, Google Cast, Sonos, IKEA Tradfri and Apple Homekit compatible devices.
- Automate everything - Make all the devices in your home work in harmony - have your lights dim when you start watching a movie, or turn off your heat when you’re away from home.
- Keep your home’s data in the home - use it privately to see past trends and averages.
- Connect to open standards with hardware add-ons - including Z-Wave, Zigbee, Matter, Thread, and Bluetooth.
- Connect Anywhere - If you would like to access this app while away from home, the most secure and simple way to get started is Home Assistant Cloud.

The app unlocks your smartphone or tablet as a home automation tool,
- Securely share your location, using it to automate heating, security, and so much more.
- Can share your phone's sensors with Home Assistant for automations including information on: steps taken, battery level, connectivity, next alarm, and many more.
- Get notifications on what's happening in your home, from detecting leaks to doors left open, you have full control over what it tells you.

> Android Auto functionality lets you open a garage or disable the security system from the dash of your car.

Reviewer: This one is missing a `- `

It also suggests you can only use covers/security systems, while you can also toggle a lot of other things, start scripts, navigate to zones/persons... Is that intentional?

> Android Auto functionality lets you open a garage or disable the security system from the dash of your car.

Author: ```suggestion
- Android Auto functionality lets you control your home from the dash of your car - open the garage, disable the security system, and more.
```
You're right its missing the `-`, and I'll make the functionality clearer
